src/pages/Home/components/BenefitsSection/BenefitsSection.js

```diff
@@ -1,8 +1,27 @@
 import { Benefit } from "./Benefit/Benefit"
+import information from './Benefit/imgs/information.png'
```

Comment:
I think it's better to name it `informationImg`, so it's clear that it's `img`, not text.

Reply:
+